### PR TITLE
craig 167: vsi form bugs

### PR DESCRIPTION
**Pull request by aisaly**
- [x] standardize lists to have same name as other forms
- [x]  change api endpoint prop names
- [x]  add securityGroups prop to VsiForm & add SecurityGroupMultiSelect into the form
- [x]  also fixed some circular dependency issues by directly referencing import files instead of the index.js

### CODE REVIEW

**Review by jvallexm — APPROVED** [review]
lgtm, just change version to 0.3.15 in package.json

File: src/components/forms/VsiForm.js [hunk 1]
```diff
@@ -110,6 +118,24 @@ class VsiForm extends Component {
               }
             />
           )}
+          <SecurityGroupMultiSelect
+            id="vsi-security-groups"
```
Comment (isaaceichelberger) [inline src/components/forms/VsiForm.js:122]: add `key={this.state.vpc + "-sg"}`

Comment (isaaceichelberger) [reply src/components/forms/VsiForm.js:122]: and do the same for subnets

**Review by isaaceichelberger — CHANGES_REQUESTED** [review]
couple small changes, ty for the dependencies too :)

**Review by isaaceichelberger — APPROVED** [review]

**Pull request merged**